openapi/kas-fleet-manager.yaml

```diff
@@ -1066,22 +1066,24 @@ components:
                   - $ref: "#/components/schemas/KafkaRequest"
     VersionMetadata:
       allOf:
-        - $ref: "#/components/schemas/ObjectReference"
-        - type: object
-          example:
-            kind: "APIVersion"
-            id: "v1"
-            href: "/api/kafkas_mgmt/v1"
-            collections:
-              - id: "kafkas"
-                href: "/api/kafkas_mgmt/v1/kafkas"
-                kind: "KafkaList"
-          properties:
-            collections:
-              type: array
-              items:
-                allOf:
-                  - $ref: "#/components/schemas/ObjectReference"
+      - $ref: "#/components/schemas/ObjectReference"
+      - type: object
+        example:
```

Comment:
The example should be updated to include server_version with an example value

Reply:
Thanks a million @miguelsorianod ! Just committed the requested change.